### PR TITLE
Acrn dm args

### PR DESCRIPTION
**Pull request by gvancuts**
Update the code (`acrn-dm`) and documentation that describe the different arguments and parameters that the ACRN Device Model (`acrn-dm`) can take. This brings them up to date with the actual implementation.

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br>No New Function Declaration/Definition Mismatch

**Review by fitchbe — APPROVED** [review]

File: doc/developer-guides/hld/hld-devicemodel.rst [hunk 1]
```diff
@@ -95,6 +95,8 @@ options:
        --logger_setting: params like console,level=4;kmsg,level=3
        --pm_notify_channel: define the channel used to notify guest about power event
        --pm_by_vuart:pty,/run/acrn/vuart_vmname or tty,/dev/ttySn
+       --windows: support Oracle virtio-blk, virtio-net and virtio-input devices
```
[the remark this answers is not in the document]
Comment (gvancuts) [reply doc/developer-guides/hld/hld-devicemodel.rst]: > virtio-blk, virtio-net and virtio-input --> virtio-blk, virtio-net, and virtio-input
> (add the serial comma)

Done

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br>No New Function Declaration/Definition Mismatch

**Review by ywan170 — APPROVED** [review]

**Comment by wenlingz** [comment]
@gvancuts  build acrn-devicemodel fail, please check:

sos/work/corei7-64-oe-linux/acrn-devicemodel/5782-1614300169/recipe-sysroot/usr/include/acrn -DASL_COMPILER=\"/usr/bin/iasl\" -fstack-protector-strong -DDM_DEBUG -c log/disk_logger.c -o /home/meta-acrn/workspace/build/master-acrn-sos/work/corei7-64-oe-linux/acrn-devicemodel/5782-1614300169/build/devicemodel/log/disk_logger.o -MMD -MT /home/meta-acrn/workspace/build/master-acrn-sos/work/corei7-64-oe-linux/acrn-devicemodel/5782-1614300169/build/devicemodel/log/disk_logger.o
core/main.c: In function 'usage':
core/main.c:141:3: error: too many arguments for format [-Werror=format-extra-args]
  141 |   "Usage: %s [-hAWYv] [-B bootargs] [-E elf_image_path]\n"
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
Makefile:231: recipe for target '/home/meta-acrn/workspace/build/master-acrn-sos/work/corei7-64-oe-linux/acrn-devicemodel/5782-1614300169/build/devicemodel/core/main.o' failed
make[1]: *** [/home/meta-acrn/workspace/build/master-acrn-sos/work/corei7-64-oe-linux/acrn-devicemodel/5782-1614300169/build/devicemodel/core/main.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/home/meta-acrn/workspace/build/master-acrn-sos/work/corei7-64-oe-linux/acrn-devicemodel/5782-1614300169/git/devicemodel'
Makefile:108: recipe for target 'devicemodel' failed
make: *** [devicemodel] Error 2
WARNING: /home/meta-acrn/workspace/build/master-acrn-sos/work/corei7-64-oe-linux/acrn-devicemodel/5782-1614300169/temp/run.do_compile.1555:1 exit 1 from 'exit 1'

ERROR: Logfile of failure stored in: /home/meta-acrn/workspace/build/master-acrn-sos/work/corei7-64-oe-linux/acrn-devicemodel/5782-1614300169/temp/log.do_compile.1555
Log data follows:

**Comment by gvancuts** [comment]
> @gvancuts build acrn-devicemodel fail, please check:

Should be fixed now... sorry about that.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br>No New Function Declaration/Definition Mismatch

**Pull request merged**